plugin/trino-phoenix5/src/test/java/io/trino/plugin/phoenix5/TestPhoenixConnectorTest.java

```diff
@@ -105,6 +106,7 @@ public void testRenameColumn()
         assertThatThrownBy(super::testRenameColumn)
                 // TODO (https://github.com/trinodb/trino/issues/7205) support column rename in Phoenix
                 .hasMessageContaining("Syntax error. Encountered \"RENAME\"");
+        throw new SkipException("Rename column is not yet supported by Phoenix connector");
```

Comment:
Do we need both the assertions and to throw the exception ?

Reply:
I like having the assertions since it makes tests automatically fail when the feature is implemented and helps ensure coverage exists instead of forgetting to re-enable tests.